models/issue.go

```diff
@@ -246,6 +246,7 @@ func (issue *Issue) APIFormat() *api.Issue {
 
 	apiIssue := &api.Issue{
 		ID:       issue.ID,
+		URL:      issue.HTMLURL(),
```

Comment:
Indeed it is the API-URL :) and that function doesn't exist, so we need a `issue.APIURL()` as well (should be fairly simple)